test/e2e/multi-select.test.js

```diff
@@ -9,8 +9,115 @@ describe('vl-multi-select', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetOnbeperkteResultaten();
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'aakstraat');
+        await multiselect.selectByValue('aakstraat');
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'aakstraat');
+    });
+
+    it('Als gebruiker kan ik een waarde uit de multiselect selecteren via text', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetOnbeperkteResultaten();
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'aalmoezenierstraat');
+        await multiselect.selectByText('Aalmoezenierstraat');
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'aalmoezenierstraat');
+    });
+
+    it('Als gebruiker kan ik een gekozen optie verwijderen', async () => {
+        const multiselect = await vlMultiSelectPage.getGegroepeerdeMultiselect();
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Bruges');
+        await multiselect.delete('Bruges');
+        await assert.eventually.notInclude(multiselect.getSelectedOptionsByValue(), 'Bruges');
+    });
+
+    it('Als gebruiker kan ik een multiselect definieren als error multiselect', async () => {
+        const multiselect = await vlMultiSelectPage.getErrorMultiselect();
+        await assert.eventually.isTrue(multiselect.isError());
+    });
+
+    it('Als gebruiker kan ik een multiselect definieren als success multiselect', async () => {
+        const multiselect = await vlMultiSelectPage.getSuccessMultiselect();
+        await assert.eventually.isTrue(multiselect.isSuccess());
+    });
+
+    it('Als gebruiker kan ik een multiselect definieren als disabled', async () => {
+        const multiselect = await vlMultiSelectPage.getDisabledMultiselect();
+        await assert.eventually.isTrue(multiselect.isDisabled());
+    });
+
+    it('Het aantal resultaten van een zoekopdracht kan beperkt worden', async () => {
+        const multiselect = await vlMultiSelectPage.getMultiselectMetSpecifiekAantalResultaten();
+        await multiselect.searchByPartialText('straat');
+        await assert.eventually.equal(multiselect.getNumberOfSearchResults(), 5);
+    });
+
+    it('Als gebruiker kan ik de values van een select opvragen', async () => {
+        const multiselect = await vlMultiSelectPage.getStandardMultiselect();
+        await assert.eventually.include(multiselect.values(), 'Belgium', 'France', 'Germany');
+    });
+
+    it('Als gebruiker kan ik controleren of een bepaalde value in de select beschikbaar is', async () => {
+        const multiselect = await vlMultiSelectPage.getStandardMultiselect();
+        await assert.eventually.isTrue(multiselect.hasValue('France'));
+    });
+
+    it('Als gebruiker kan ik luisteren naar een change event en onChange loggen naar de console', async () => {
```

Comment:
Beschrijving klopt niet meer met de test: loggen naar console. Best ook beschrijving van de demo pagina aanpassen of de console.log ook nog laten uitvoeren ter info.

Reply:
Had ik al aangepast :)